README.md

```diff
@@ -36,7 +36,10 @@ packages:
 3. Add an on-run-end hook to your `dbt_project.yml`: `on-run-end: "{{ dbt_artifacts.upload_results(results) }}"`
 (We recommend adding a conditional here so that the upload only occurs in your production environment, such as `on-run-end: "{% if target.name == 'prod' %}{{ dbt_artifacts.upload_results(results) }}{% endif %}"`)
 
-4. Create the tables dbt_artifacts uploads to with `dbt run-operation create_dbt_artifacts_tables`
+4. In case you don't want the artifacts to be uploaded for certain dbt commands (for instance 'dbt compile'), add the following conditional:
+`on-run-end: "{% if not var('dbt_artifacts_upload_results', False) %}{{ dbt_artifacts.upload_results(results) }}{% endif %}"`
+If you then run the following command by providing a variable to the command, the 'on-run-end' will not execute:
+`dbt compile --vars 'artifact: disable'`
```

Comment:
Going to remove this for now but feel free to add to a separate PR @charles-astrafy

Reply:
Sure, will put it in a separate PR.